src/hooks/dialogmote/useInitialValuesReferat.ts

```diff
@@ -18,7 +18,7 @@ export const useInitialValuesReferat = (
     arbeidsgiver: { virksomhetsnummer },
     behandler,
   } = dialogmote;
-  const { latestReferat } = useDialogmoteReferat(dialogmote);
+  const { latestReferat } = getDialogmoteReferat(dialogmote);
```

Comment:
Trodde `useDialogmoteReferat` var en hook utifra navnet, så loket lenge med det, men er egentlig bare en funksjon for å hente ut eventuelle referat fra et dialogmøte 🙃